### ISSUE TITLE
VFS tools repo - Accessibility Package

### ISSUE THREAD

**Issue by oseasmoran73** [department-of-veterans-affairs/va.gov-team#30517]
## Description

Determine why the accessibility package cannot be installed on linux

**Comment by oseasmoran73** [comment]
Ticket spun up from [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/2259)

**Comment by timwright12** [comment]
closing this out because I don't think this package exists
